### ISSUE TITLE
Add support for TDS Tokens (TABNAME, COLINFO)

### ISSUE THREAD

**Issue by tracker1** [tediousjs/tedious#242]
Per the email chain with Microsoft regarding #238 ...

<blockquote>... I can at least notice that it doesn’t handle TDS tokens TABNAME and COLINFO. I suspect there are other shortcomings. -- <cite>Yosef Dinerstein &lt;yosefd at microsoft.com&gt;</cite></blockquote>

**Comment by arthurschreiber** [comment]
Yeah, there's quite a bunch of things in the TDS protocol that are not implemented in Tedious, most likely because they were not used by any of the current maintainers.

Tedious does not support the `BROWSE MODE` nor things like `sp_cursoropen` and friends at the moment.
